README.md

```diff
@@ -187,6 +189,17 @@ This render prop wraps its return value in a list item (`<li>`), and receives a
 | i           | Number  | The index of the current item being iterated over.                                                             |
 | item        | Any     | The current item being iterated over, as defined by the Array fed to the `<Gallery>` component's `items` prop. |
 
+### &lt;GalleryItem&gt;
```

Comment:
Can we also add this to the table of contents at the top of the README?

Reply:
Done!